### ISSUE TITLE
Homepage Design

### ISSUE THREAD

**Issue by gabrielfdac** [OSLC/oslc-site-hugo#7]
https://xd.adobe.com/view/f5a37d61-fe4c-4a78-a114-b75c9cc2010c

This is the current design of the homepage. We tried to keep it as simple and modern as possible.
One thing that I believe we can definitely improve is with a better fitting image for the Hero (section just under the header).

Please share your feedback.

![image](https://user-images.githubusercontent.com/16785479/27845137-9684c638-60f1-11e7-9325-7941de1836e7.png)
_Here is an alternative to the hero section, without an image_

**Comment by brianking** [comment]
This is great. Clean and clear. A comment and question:

1. The grouping of the About and Contribute sections together works really well. You can have both in view together, and it caters to new and return visitors well.

2. I see that some sections are centered and do not stretch the full width. Is there a reason for that?

**Comment by gabrielfdac** [comment]
Thanks for the feedback, here I address them and add a question:

1. Thanks, I thought so as well

2. Yes there is. This is a good practice for webpages for users with big resolutions (larger than 1600 pixels wide). To use a "container" to limit how large regular content can be.

The reason is that if the context, specially text, spreads too wide along the width of the screen if becomes hard to read and visualize. The exception of that is if you can somehow section the content, like the blog cards, or the resources columns for example. In those cases it's okay to use the whole screen since the content within is contained. 

On this design we're using a "base" resolution of 1920 x 1080. On a smaller screen there won't be as much "empty space" on the sides as we're using a container of 1024px. So in the max resolution of a 13" monitor (usually 1360 x 760) to content will feel as if it contains more of the screen.

To illustrate why that is a good practice I manipulated one of Apples pages on the console to show the comparison between the two cases:

_This is the edited version_
![image](https://user-images.githubusercontent.com/16785479/27874259-817390ca-6174-11e7-913e-0a7319f0bc50.png)

_Here is the original_
![image](https://user-images.githubusercontent.com/16785479/27874292-9ed25020-6174-11e7-8551-355b5b49d8bf.png)

As you can see on a larger screen, having the content "spread" too much is actually quite bad for the reader as it becomes hard to follow the whole content. The containers usually follow a similar size of a book page, which is about the perfect size for reading.

Even this page on my screen, which is 1920 x 1080, has all the content centralized.
![image](https://user-images.githubusercontent.com/16785479/27874719-20e66f0a-6176-11e7-9a48-de9ca20b37ff.png)

I hope this helps to illustrate why it's important to have it that way, and in case you use a smaller monitor I can share a 1360 x 760 version so that it would better represent how you or someone else with a smaller screen would see the content.

3. How about the covers? Which of the two you prefer? Also if you have an image you'd like or want to try just let me know.

**Comment by brianking** [comment]
Thanks for the explanation. Of course we'll need the site to be responsive, so this will help.

For the cover, I like both. Maybe for now we can just use the alternative one you have with just colours.

**Comment by berezovskyi** [comment]
Looks great! I think we need to work on the gray "landing" section, I understand that this is just a dummy.

I was thinking we would need to address at least 3 stakeholders, maybe use something like this:

![image](https://user-images.githubusercontent.com/64734/27906007-1b046d90-6242-11e7-92a0-16218aa095ce.png)

or if we have more content, then the simple approach of a section followed by a button from http://getbootstrap.com/ might be better.

**Comment by gabrielfdac** [comment]
I just updated the link with the alternative Hero, and even before reading your comment Andrew I updated the color of the grey as I noticed it was a bit too dark for what I was going for.

If the content is to remain similar, as a brief About followed by Contribute, I think it would be better to just keep them as sections. Content side-by-side like that suggest a correlation between them.

What we could do, as there's different ways to contribute instead of describing them as text we could have the approach you suggested. Say one column for "Develop", "Usergroups" and "Content" or what suits best. 

It all depends on what content we want to show on the homepage.

However this approach is shifts the focus heavily on to contributors, focusing that much of the homepage on that content is naturally going to attract those more than regular users. So my suggestion if we decide to have such approach is to have the resources section before this one, as it will cater to the new comers better and those are the one you generally need to pull the more attention from.

As for keeping them as sections, the current design is pretty much the same as the bootstrap example, or do you mean the button style?

**Comment by brianking** [comment]
@berezovskyi and I chatted today and he clarified his point about the section for stakeholders. So, this will actually be a different section that we originally planned, and what I somehow overlooked when planning (or didn't get copied to the issues). 

Basically, we need a section on the site with case studies that show the value of OSLC technologies to possible stakeholders. I opened issue #28 which covers in general what site features we need for this. For the purpose of the homepages, we need something like what is outlined in Andrew's comment that has lead-in sections for 3 audiences.

The audiences we are targeting are:

* Vendors - why should current tools vendors support OSLC, and why is it complimentary to their business and not a threat
* Users (i.e. mostly companies) / System Integrators - why OSLC won't disrupt their workflow (continue using your datasources), and value added on top
* Developers

**Comment by gabrielfdac** [comment]
I just updated the Link Above with a Case Studies Section. Let me know your thoughts

**Comment by brianking** [comment]
RE the case studies section, the design looks good. However, I think it needs to be differentiated more from other sections. So here is what I suggest:

* Give the section a title. It could be just 'Case Studies', or something to grab people's attention a bit more like 'How OSLC Is For You'
* Add some lead in text
* Add 'Learn More' buttons
* Consider changing the background colour for contrast, but might not be necessary. I will defer to your design opinion on this

So, it is very similar to the section that @berezovskyi posted a picture of earlier in this issue, which can be found at http://foundation.zurb.com/

**Comment by brianking** [comment]
Any thoughts on my last comment @gabrielfdac ?

**Comment by brianking** [comment]
TODO: @gabrielfdac to update the Case Studies section.

**Comment by brianking** [comment]
Two outstanding items here:

* Add some lead in text
* Add 'Learn More' buttons

**Comment by brianking** [comment]
Closing this now. We can open new issues for whatever is outstanding.
